### ISSUE TITLE
bash shell style settings

### ISSUE THREAD

**Issue by RolandHughes** [atom/language-shellscript#12]
When auto-indent is turned on editing a bash shell the editor seems to be enforcing an archaic style.
```bash
if [ -f some_file ]; then
   do something
fi
```

Vast majority of shops I work with only have extremely old legacy code with that style as all new/edited code must adhere to modern style guides which mandate:

```bash
if [ -f some_file]
then
    do something
fi
```

Most other editors I have used, if they support a file extension based style, have some method of tweaking said style or selecting between base styles. Some editors have multiple base styles for different languages, then let you customize after choosing (the C language usually has this.)

Could we either have a choice between modern and archaic/legacy styles with shell scripts, if not the ability to tweak a style and load it on all of the machines in an office?

**Comment by thomasjo** [comment]
I updated your issue body to use fenced code blocks, hope you don't mind :bowtie: 

As for your actual issue, this is something I wish we had support for, but that we currently don't. Hopefully someone will contribute a PR, publish a community package or some such that at least partially resolves your enhancement idea.

**Comment by 50Wliu** [comment]
Should be fixed by #30 and will be available in Atom 1.2.0.
